### PR TITLE
 User-story: 11547

### PR DESCRIPTION
**Pull request by ws-mps**
Extend stock.picking._backorder_movelines()

Added optional parameter `picking` so that move lines can be moved
into an existing picking instead of creating a new one.

### CODE REVIEW

File: addons/udes_stock/models/stock_picking.py [hunk 1]
```diff
@@ -907,41 +907,44 @@ def _requires_backorder(self, mls):
                 return True
         return False
 
-    def _backorder_movelines(self, mls=None):
-        """ Creates a backorder pick from self (expects a singleton)
+    def _backorder_movelines(self, mls=None, picking=None):
```
Comment (squah-u) [inline addons/udes_stock/models/stock_picking.py:910]: nit: Can we name it `dest_picking` or something that's not `picking`, to distinguish it from the `self` picking?

**Review by squah-u — APPROVED** [review]

**Review by nocturnalastro — APPROVED** [review]

File: addons/udes_stock/models/stock_picking.py (hunk 1 above)
Comment (ws-mps) [reply addons/udes_stock/models/stock_picking.py]: Done, please review.

Comment (squah-u) [reply addons/udes_stock/models/stock_picking.py]: The final `return` needs updating

Comment (ws-mps) [reply addons/udes_stock/models/stock_picking.py]: Changed too :)

Comment (squah-u) [reply addons/udes_stock/models/stock_picking.py]: lgtm

**Review by williamsajr — APPROVED** [review]

**Review by apatrick88 — APPROVED** [review]

**Pull request merged**